workflow/rocoto/workflow_tasks.py

```diff
@@ -1259,9 +1259,10 @@ def epos(self):
         def _get_eposgroups(epos):
             fhmin = epos['FHMIN_ENKF']
             fhmax = epos['FHMAX_ENKF']
+            fhout = epos['FHOUT_ENKF']
             if self.cdump == "gfs":
                 fhmax = epos['FHMAX_ENKF_GFS']
-            fhout = epos['FHOUT_ENKF']
+                fhout = epos['FHOUT_ENKF_GFS']
```

Comment:
If you make it `FHOUT_ENKF_gfs` in `config.base`, be use to make the same change here.

Reply:
Thanks. As mentioned in the previous answer, I think it is better to use FHOUT_ENKF_GFS since there is no variable FHOUT_ENKF_gdas. And also consistent with other variables, such as FHMAX_ENKF_GFS. How do you think?